### ISSUE TITLE
About the loss function in the code

### ISSUE THREAD

**Issue by destructive-observer** [yinboc/liif#19]
Thanks a lot for your excellent work.But i get a confusion in the code.The loss function is L1-loss in  your work, but i can not find the loss for the edsr-baseline.Is it only a L1-Loss to train for your mlp-layers and your edsr-baseline as a whole networks? Forward your reply ,thank you.

**Comment by yinboc** [comment]
Thanks for your interest. Yes, there is only a L1 loss between the final output and ground-truth.
